src/content/store-and-form-setup.md

```diff
@@ -26,27 +38,46 @@ export interface IBioSummary {
 
 ---
 
+## Reduce, reuse...
+
+Plan any other forms you want to add to your form state
+
+```ts
+export interface IForm {
+  character: ICharacter; // state.form.character
+  equipment: IEquipment; // state.form.equipment
+}
+
+export interface IEquipment {
+  weaponName?: string;
+  weaponType: string;
+  armorType: string;
+}
+```
+
+---
+
 ## Setting up actions
```

Comment:
What do you think about something like:
`Setting up actions (you don't wanna be surprised)`
or
`Setting up your spells, ops.. actions!`